src/components/MyWorkOrders.js

```diff
@@ -44,14 +45,23 @@ class MyWorkOrders extends Component {
           <h1>
             <FontAwesomeIcon icon={faStreetView} /> My Work Orders
           </h1>
+          <div className="d-flex flex-row">
```

Comment:
I'm wondering if we should create a standardized button component that takes certain props like: 
```jsx
<Button 
  spacingModifierClasses="mr-2 mb-2" 
  linkPath="/work-order/new/"
  faIcon={faWrench}
  text="New Work Order"
/>
```

And the Button component could also have defaults for all those props too. This could save us some energy when we resize the buttons soon and ensure they are pretty uniform. So yeah, maybe we could save this idea for that task when it comes up...

Reply:
Agreed - especially since I am about to add another button to this view for Inventory Items.